includes/notifications.php

```diff
@@ -58,15 +58,16 @@ function notification_screen() {
 
 	$output = $unagi_nags;
 
+	$output = '<h2>' . esc_html__( 'Notifications', 'unagi' ) . '</h2>';
+
 	if ( show_notifications_nicely() ) {
 		$notification_info = prepare_notification_info();
 
-		$output = $notification_info['content'];
-
+		$output .= $notification_info['content'];
 	}
 
-	if ( empty( $output ) ) {
-		$output = sprintf(
+	if ( empty( $notification_info['content'] ) ) {
```

Comment:
`$notification_info['content']`  can throw a warning when `show_notifications_nicely()` is false

Reply:
I'm not sure why would that throw a warning, on which PHP?

Reply:
`$notification_info` is inaccessible scope so `$notification_info['content']` too.

Reply:
@mustafauysal I'm not sure about this and the proposal below would solve this anyway, but the docs reads:

> empty() does not generate a warning if the variable does not exist.